### PR TITLE
Add test case for upgrade for migrated Longhorn on Rancher

### PR DESCRIPTION
**Pull request by khushboo-rancher**
Add test case for Longhorn upgrade on Rancher where the Longhorn app was migrated from cluster explorer to dashboard

Signed-off-by: Khushboo <fnu.khushboo@suse.com>

### CODE REVIEW

**Comment by khushboo-rancher** [comment]
@PhanLe1010  Could you please update the branch `release-v2.6-longhorn-1.3.1` in your repo to have the broken chart longhorn 100.2.1+up1.3.1.

**Comment by PhanLe1010** [comment]
> @PhanLe1010 Could you please update the branch release-v2.6-longhorn-1.3.1 in your repo to have the broken chart longhorn 100.2.1+up1.3.1.

Done. Thank you

**Review by PhanLe1010 — APPROVED** [review]
LGTM

File: docs/content/manual/release-specific/v1.4.0/test-upgrade-for-migrated-longhorn.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+https://github.com/longhorn/longhorn/discussions/4198
+
+**Context**: since few customers used our broken chart longhorn 100.2.1+up1.3.1 on Rancher (Now fixed) with the workaround. We would like to verify the future upgrade path for those customers.
+
+## Steps
+
+1. Set up a cluster of Kubernetes 1.20.
+1. Adding this repo to the apps section in new rancher UI
+    1. repo: https://github.com/PhanLe1010/charts.git
+    1. branch: release-v2.6-longhorn-1.3.1.
```
Comment (innobead) [inline docs/content/manual/release-specific/v1.4.0/test-upgrade-for-migrated-longhorn.md:15]: This branch from Phan is intended :)

**Review by innobead — APPROVED** [review]

**Pull request merged**